client/src/components/NewVersionAvailable.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Dialog from './Dialog';
+import css from './NewVersionAvailable.css';
+
+const NewVersionAvailable = ({ newVersionAvailable }) => (
+  <Dialog
+    onClose={() => { window.location.href = '.'; }}
+    title="En ny versjon av applikasjonen er tilgjengelig"
+    visible={newVersionAvailable}
+  >
+    <p>
+      <a href="." className={css.reloadPageLink}>
+        Klikk her for å laste inn siden på nytt.
```

Comment:
Ideally I'd see this as a link covering the whole of card content, but that isn't really possible with our Card component. I'd maybe go with a button instead of anchor text, could use a ButtonIconText with background. This is mere personal preference though.